### PR TITLE
Minor style fix

### PR DESCRIPTION
**Pull request by xiaoxiang781216**
### Description of the feature or fix

- fix(misc): avoid ;; after LV_ASSERT_STYLE/LV_ASSERT_OBJ
- fix(style_api_gen.py): remove the unused docs_prop_cnt variable
- fix(style_api_gen.py): ensure props has the same order of lv_style_prop_t

### Checkpoints
- [X] Follow the [styling guide](https://github.com/lvgl/lvgl/blob/master/docs/CODING_STYLE.md)
- [ ] Update [CHANGELOG.md](https://github.com/lvgl/lvgl/blob/master/docs/CHANGELOG.md)
- [ ] Update the documentation

### CODE REVIEW

File: docs/overview/style-props.md [hunk 1]
```diff
@@ -3,901 +3,743 @@
...
 
-
 ### width
 Sets the width of object. Pixel, percentage and `LV_SIZE_CONTENT` values can be used. Percentage values are relative to the width of the parent's content area.
 <ul>
-<li style='display:inline; margin-right: 20px; margin-left: 0px'><strong>Default</strong> widget dependent</li>
-<li style='display:inline; margin-right: 20px; margin-left: 0px'><strong>Inherited</strong> No</li>
-<li style='display:inline; margin-right: 20px; margin-left: 0px'><strong>Layout</strong> Yes</li>
-<li style='display:inline; margin-right: 20px; margin-left: 0px'><strong>Ext. draw</strong> No</li>
+<li style='display:inline; margin-right: 20px; margin-left: 0px'><strong>Default</strong>Widget dependent</li>
```
Comment (kisvegabor) [inline docs/overview/style-props.md]: There should be a space after `</strong>`:
```html
<strong>Default</strong> Widget dependent
```

**Review by kisvegabor — COMMENTED** [review]
Thanks, I noticed a minor formatting  issue.

File: docs/overview/style-props.md (hunk 1 above)
Comment (xiaoxiang781216) [reply docs/overview/style-props.md]: It's strange that: there is one space before "Widget dependent", but no space after it. So, I change the format. Do you think so @kisvegabor ?

Comment (kisvegabor) [reply docs/overview/style-props.md]: The space after the items is created with `margin-right:20px` of the `<li>` elements.

Comment (xiaoxiang781216) [reply docs/overview/style-props.md]: Hmm, understand. Fixed, @kisvegabor

**Comment by kisvegabor** [comment]
Looks good, thank you!

**Pull request merged**